packages/forms-web-app/src/controllers/lpa-dashboard/your-appeals.js

```diff
@@ -24,17 +22,31 @@ const getYourAppeals = async (req, res) => {
 
 	appealsCaseData = await getAppealsCaseData(user.lpaCode);
 
-	appealsCaseData.forEach((appeal) => {
-		appeal.dueInDays = calculateDueInDays(appeal.questionnaireDueDate);
-		appeal.appealNumber = extractAppealNumber(appeal.caseReference);
-		appeal.address = formatAddress(appeal);
-		appeal.appealType = formatAppealType(appeal.appealType);
-	});
+	const { toDoAppeals, waitingForReviewAppeals } = appealsCaseData
+		.map(mapToLPADashboardDisplayData)
+		.reduce(
+			(acc, cur) => {
```

Comment:
I don't think we need to recreate the object + arrays each time. A simple push will suffice and be clearer.  What d'ya reckon?

```
if (islpa...) {
  acc.toDoAppeals.push(cur);
} else {
  acc.waitingForReviewAppeals.push(cur);
}
return acc;
``